Description: fix typos in messages written to stdout and/or log files
Forwarded: yes
Author: Pau Garcia i Quiles <pgquiles@elpauer.org>
Last-Update: 2014-06-29

diff -rupd ACE_wrappers.orig/ace/Dev_Poll_Reactor.cpp ACE_wrappers/ace/Dev_Poll_Reactor.cpp
--- ACE_wrappers.orig/ace/Dev_Poll_Reactor.cpp	2014-05-22 20:41:22.000000000 +0200
+++ ACE_wrappers/ace/Dev_Poll_Reactor.cpp	2014-06-29 17:32:00.519614000 +0200
@@ -2284,7 +2284,7 @@ ACE_Dev_Poll_Reactor::mask_ops_i (ACE_HA
     {
     case ACE_Reactor::GET_MASK:
       // The work for this operation is done in all cases at the
-      // begining of the function.
+      // beginning of the function.
       return old_mask;
 
     case ACE_Reactor::CLR_MASK:
diff -rupd ACE_wrappers.orig/ace/Get_Opt.cpp ACE_wrappers/ace/Get_Opt.cpp
--- ACE_wrappers.orig/ace/Get_Opt.cpp	2013-04-11 17:50:32.000000000 +0200
+++ ACE_wrappers/ace/Get_Opt.cpp	2014-06-29 17:31:18.839614000 +0200
@@ -130,7 +130,7 @@ ACE_Get_Opt::ACE_Get_Opt (int argc,
     this->ordering_ = REQUIRE_ORDER;
 
   // Now, check to see if any or the following were passed at
-  // the begining of optstring: '+' same as POSIXLY_CORRECT;
+  // the beginning of optstring: '+' same as POSIXLY_CORRECT;
   // '-' turns off POSIXLY_CORRECT; or ':' which signifies we
   // should return ':' if a parameter is missing for an option.
   // We use a loop here, since a combination of "{+|-}:" in any
diff -rupd ACE_wrappers.orig/ace/MMAP_Memory_Pool.cpp ACE_wrappers/ace/MMAP_Memory_Pool.cpp
--- ACE_wrappers.orig/ace/MMAP_Memory_Pool.cpp	2013-09-01 02:58:08.000000000 +0200
+++ ACE_wrappers/ace/MMAP_Memory_Pool.cpp	2014-06-29 17:36:45.399614000 +0200
@@ -476,7 +476,7 @@ ACE_MMAP_Memory_Pool_Options::ACE_MMAP_M
     install_signal_handler_ (install_signal_handler)
 {
   ACE_TRACE ("ACE_MMAP_Memory_Pool_Options::ACE_MMAP_Memory_Pool_Options");
-  // for backwards compatability
+  // for backwards compatibility
   if (base_addr_ == 0 && use_fixed_addr_ == ALWAYS_FIXED)
     use_fixed_addr_ = FIRSTCALL_FIXED;
 }
diff -rupd ACE_wrappers.orig/ace/Select_Reactor_Base.cpp ACE_wrappers/ace/Select_Reactor_Base.cpp
--- ACE_wrappers.orig/ace/Select_Reactor_Base.cpp	2013-08-07 09:10:20.000000000 +0200
+++ ACE_wrappers/ace/Select_Reactor_Base.cpp	2014-06-29 17:31:35.447614000 +0200
@@ -1013,7 +1013,7 @@ ACE_Select_Reactor_Impl::bit_ops (ACE_HA
     {
     case ACE_Reactor::GET_MASK:
       // The work for this operation is done in all cases at the
-      // begining of the function.
+      // beginning of the function.
       break;
     case ACE_Reactor::CLR_MASK:
       ptmf = &ACE_Handle_Set::clr_bit;
diff -rupd ACE_wrappers.orig/ace/Service_Gestalt.cpp ACE_wrappers/ace/Service_Gestalt.cpp
--- ACE_wrappers.orig/ace/Service_Gestalt.cpp	2013-04-11 17:50:32.000000000 +0200
+++ ACE_wrappers/ace/Service_Gestalt.cpp	2014-06-29 17:31:48.743614000 +0200
@@ -60,7 +60,7 @@ ACE_Service_Type_Dynamic_Guard::ACE_Serv
     if (ACE::debug ())
         ACELIB_DEBUG ((LM_DEBUG,
                     ACE_TEXT ("ACE (%P|%t) STDG::<ctor>, repo=%@")
-                    ACE_TEXT(", name=%s - begining at [%d]\n"),
+                    ACE_TEXT(", name=%s - beginning at [%d]\n"),
                     &this->repo_,
                     this->name_,
                     this->repo_begin_));
diff -rupd ACE_wrappers.orig/ace/Sock_Connect.cpp ACE_wrappers/ace/Sock_Connect.cpp
--- ACE_wrappers.orig/ace/Sock_Connect.cpp	2013-12-02 19:33:22.000000000 +0100
+++ ACE_wrappers/ace/Sock_Connect.cpp	2014-06-29 17:33:11.959614000 +0200
@@ -314,7 +314,7 @@ ACE::get_bcast_addr (ACE_UINT32 &bcast_a
         ACELIB_ERROR ((LM_ERROR,
                     ACE_TEXT ("%p\n"),
                     ACE_TEXT ("ACE::get_bcast_addr:")
-                    ACE_TEXT ("Broadcast is not enable for this interface.")));
+                    ACE_TEXT ("Broadcast is not enabled for this interface.")));
 
       if (handle == ACE_INVALID_HANDLE)
         ACE_OS::close (s);
diff -rupd ACE_wrappers.orig/ace/SOCK_Dgram_Bcast.cpp ACE_wrappers/ace/SOCK_Dgram_Bcast.cpp
--- ACE_wrappers.orig/ace/SOCK_Dgram_Bcast.cpp	2013-04-11 17:50:32.000000000 +0200
+++ ACE_wrappers/ace/SOCK_Dgram_Bcast.cpp	2014-06-29 17:33:26.807614000 +0200
@@ -260,7 +260,7 @@ ACE_SOCK_Dgram_Bcast::mk_broadcast (cons
         {
           if (host_name != 0)
             ACELIB_ERROR ((LM_ERROR, ACE_TEXT("%p [%s]\n"),
-                        ACE_TEXT("ACE_SOCK_Dgram_Bcast::mk_broadcast: Broadcast is not enable for this interface."),
+                        ACE_TEXT("ACE_SOCK_Dgram_Bcast::mk_broadcast: Broadcast is not enabled for this interface."),
                         flags.ifr_name));
         }
     }
diff -rupd ACE_wrappers.orig/ace/WFMO_Reactor.cpp ACE_wrappers/ace/WFMO_Reactor.cpp
--- ACE_wrappers.orig/ace/WFMO_Reactor.cpp	2014-03-07 13:01:21.000000000 +0100
+++ ACE_wrappers/ace/WFMO_Reactor.cpp	2014-06-29 17:31:04.639614000 +0200
@@ -173,7 +173,7 @@ ACE_WFMO_Reactor_Handler_Repository::bit
     case ACE_Reactor::GET_MASK:
 
       // The work for this operation is done in all cases at the
-      // begining of the function.
+      // beginning of the function.
 
       ACE_UNUSED_ARG (change_masks);
 
diff -rupd ACE_wrappers.orig/bin/fuzz.pl ACE_wrappers/bin/fuzz.pl
--- ACE_wrappers.orig/bin/fuzz.pl	2013-11-22 09:24:27.000000000 +0100
+++ ACE_wrappers/bin/fuzz.pl	2014-06-29 17:36:31.007614000 +0200
@@ -2190,12 +2190,12 @@ sub check_for_TAO_Local_RefCounted_Objec
 }
 
 # This test checks for the correct use of ORB_init() so as
-# to be compatiable with wide character builds.
+# to be compatible with wide character builds.
 sub check_for_ORB_init ()
 {
     return if is_suppressed ();
 
-    print "Running the ORB_init() wide character incompatability check\n";
+    print "Running the ORB_init() wide character incompatibility check\n";
     foreach $file (@files_cpp, @files_inl) {
         if (open (FILE, $file)) {
             my $disable = 0;
diff -rupd ACE_wrappers.orig/MPC/docs/USAGE ACE_wrappers/MPC/docs/USAGE
--- ACE_wrappers.orig/MPC/docs/USAGE	2013-11-11 18:31:31.000000000 +0100
+++ ACE_wrappers/MPC/docs/USAGE	2014-06-29 17:33:43.943614000 +0200
@@ -202,7 +202,7 @@ overrides the template input file for al
 The -feature_file specifies a file to be read that enables or disables
 features.  These feature names can be anything, but they should correspond
 to values used for the 'requires' and 'avoids' keywords.  If a feature is
-required and is not enable then the project will not be created.  If a
+required and is not enabled then the project will not be created.  If a
 feature is to be avoided and it is enabled then the project will not be
 created.
 
